start_admin_sshd.sh

```diff
@@ -8,36 +8,86 @@ mkdir -p /home/ec2-user/.ssh/
 chmod 700 /home/ec2-user/.ssh/
 ssh_host_key_dir="/.bottlerocket/host-containers/admin/etc/ssh"
 ssh_config_dir="/home/ec2-user/.ssh"
+user_data="/.bottlerocket/host-containers/admin/user-data"
```

Comment:
nit: this, and the above, are constants - they should be declared read only:
 
```suggestion
declare -r user_data="/.bottlerocket/host-containers/admin/user-data"
```